commands/mod/delete.js

```diff
@@ -0,0 +1,78 @@
+const Command = require('../../structures/Command');
+
+const { Collection } = require('discord.js');
```

Comment:
Unused variable?

Reply:
Yeah, old version of it i used it, again this WIP 👀